<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helloClass</class>
 <widget class="QMainWindow" name="helloClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>塑管检测系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hello.qrc">
    <normaloff>:/logo/Resources/fivision.ico</normaloff>:/logo/Resources/fivision.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsLeft">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>490</width>
         <height>545</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGraphicsView" name="LeftPicView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContentsRight">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>489</width>
         <height>545</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QGraphicsView" name="RightPicView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="resizeAnchor">
           <enum>QGraphicsView::NoAnchor</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> 关于</string>
    </property>
    <addaction name="About"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>15000</width>
     <height>48</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>48</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Open"/>
   <addaction name="OnRun"/>
   <addaction name="separator"/>
   <addaction name="Configure"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="Configure">
   <property name="icon">
    <iconset>
     <normalon>:/icon/Resources/icon/01.ico</normalon>
     <activeon>:/icon/Resources/icon/02.ico</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="OnRun">
   <property name="icon">
    <iconset resource="hello.qrc">
     <normaloff>:/icon/Resources/icon/15.ico</normaloff>
     <normalon>:/icon/Resources/icon/15.ico</normalon>
     <activeoff>:/icon/Resources/icon/16.ico</activeoff>
     <activeon>:/icon/Resources/icon/16.ico</activeon>:/icon/Resources/icon/15.ico</iconset>
   </property>
   <property name="text">
    <string>单次</string>
   </property>
  </action>
  <action name="Open">
   <property name="icon">
    <iconset>
     <normalon>:/icon/Resources/icon/13.ico</normalon>
     <activeon>:/icon/Resources/icon/14.ico</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="About">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>关于……</string>
   </property>
   <property name="toolTip">
    <string>关于……</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hello.qrc"/>
 </resources>
 <connections/>
</ui>
